src/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs

```diff
@@ -71,6 +63,79 @@ public void SslStream_StreamToStream_Authentication_IncorrectServerName_Fail()
             }
         }
 
+        [Fact]
+        public void SslStream_StreamToStream_Back2Back_ClientWrite_Success()
+        {
+            SslStream clientSSLStream = null;
+            SslStream ServerSSLStream = null;
+
+            try
+            {
+                GetServerClientSSLStream(out clientSSLStream, out ServerSSLStream);
+                clientSSLStream.Write(sampleMsg);
+                clientSSLStream.Write(sampleMsg);
+            }
```

Comment:
since, I am assuming it to be a white box validation & this testcase was mainly to cover the fix, I purposefully chose not to verify the data. I can extend it though as there is no harm in extra validation :)